publish_frequency: 10
frame_id: "map"

## walls: [x1,y1,z1, x2,y2,z2]
objectID_1: [10,4,0, 5,4,0]
# steps: 
objectID_2: [3,6,0, 2,6,0]
# tables / desks [x1,y1,z1, ...]
objectID_7: [2,2,0,2,3,0,3,3,0,3,2,0, 5,2,0,5,3,0,6,3,0,6,2,0]
# humans: [x,y,z,r]
objectID_8: [0,4,0, 0.5]

